packages/svelte/src/internal/client/dom/operations.js

```diff
@@ -45,14 +41,15 @@ export function init_operations() {
 	// @ts-expect-error
 	element_prototype.__click = undefined;
 	// @ts-expect-error
-	text_prototype.__nodeValue = ' ';
-	// @ts-expect-error
 	element_prototype.__className = '';
 	// @ts-expect-error
 	element_prototype.__attributes = null;
 	// @ts-expect-error
 	element_prototype.__e = undefined;
 
+	// @ts-expect-error
+	text_prototype.__nodeValue = ' ';
```

Comment:
You can also remove the variable here and just set it on `Text.prototype.__nodeValue = ' '`

Reply:
ah, good catch